common/src/main/java/com/gregtechceu/gtceu/common/data/GTItems.java

```diff
@@ -925,73 +925,73 @@ public static ICustomDescriptionId cellName() {
-    public static ItemEntry<Item> COVER_DIGITAL_INTERFACE = REGISTRATE.item("cover.digital", Item::new).lang("Digital Interface").register();
-    public static ItemEntry<Item> COVER_DIGITAL_INTERFACE_WIRELESS = REGISTRATE.item("cover.digital.wireless", Item::new).lang("Wireless Digital Interface").register();
-    public static ItemEntry<Item> COVER_FLUID_VOIDING = REGISTRATE.item("cover.fluid.voiding", Item::new).lang("Fluid Voiding Cover").register();
-    public static ItemEntry<Item> COVER_FLUID_VOIDING_ADVANCED = REGISTRATE.item("cover.fluid.voiding.advanced", Item::new).lang("Advanced Fluid Voiding Cover").register();
-    public static ItemEntry<Item> COVER_ITEM_VOIDING = REGISTRATE.item("cover.item.voiding", Item::new).lang("Item Voiding Cover").register();
-    public static ItemEntry<Item> COVER_ITEM_VOIDING_ADVANCED = REGISTRATE.item("cover.item.voiding.advanced", Item::new).lang("Advanced Item Voiding Cover").register();
-
-    public static ItemEntry<ComponentItem> COVER_FACADE = REGISTRATE.item("cover.facade", ComponentItem::create)
+    public static ItemEntry<Item> COVER_ENDER_FLUID_LINK = REGISTRATE.item("ender_fluid_link_cover", Item::new).lang("Ender Fluid Link").register();
+    public static ItemEntry<Item> COVER_DIGITAL_INTERFACE = REGISTRATE.item("digital_interface_cover", Item::new).lang("Digital Interface").register();
+    public static ItemEntry<Item> COVER_DIGITAL_INTERFACE_WIRELESS = REGISTRATE.item("wireless_digital_interface_cover", Item::new).lang("Wireless Digital Interface").register();
+    public static ItemEntry<Item> COVER_FLUID_VOIDING = REGISTRATE.item("fluid_voiding_cover", Item::new).lang("Fluid Voiding Cover").register();
+    public static ItemEntry<Item> COVER_FLUID_VOIDING_ADVANCED = REGISTRATE.item("advanced_fluid_voiding_cover", Item::new).lang("Advanced Fluid Voiding Cover").register();
+    public static ItemEntry<Item> COVER_ITEM_VOIDING = REGISTRATE.item("item_voiding_cover", Item::new).lang("Item Voiding Cover").register();
+    public static ItemEntry<Item> COVER_ITEM_VOIDING_ADVANCED = REGISTRATE.item("advanced_item_voiding_cover", Item::new).lang("Advanced Item Voiding Cover").register();
+
+    public static ItemEntry<ComponentItem> COVER_FACADE = REGISTRATE.item("facade_cover", ComponentItem::create)
             .lang("Cover Facade")
             .onRegister(attach(new FacadeItemBehaviour(), new CoverPlaceBehavior(GTCovers.FACADE)))
             .model(NonNullBiConsumer.noop())
             .register();
 
     // Solar Panels: ID 331-346
-    public static ItemEntry<ComponentItem> COVER_SOLAR_PANEL = REGISTRATE.item("cover.solar.panel", ComponentItem::create).lang("Solar Panel").onRegister(attach(new TooltipBehavior(lines -> {
+    public static ItemEntry<ComponentItem> COVER_SOLAR_PANEL = REGISTRATE.item("solar_panel_cover", ComponentItem::create).lang("Solar Panel").onRegister(attach(new TooltipBehavior(lines -> {
```

Comment:
anything else doesn't have the `_cover` suffix. Why do solar panels?

Reply:
All the covers have `_cover` in the ID but are redacted from the in-game language name.

Reply:
the pumps, conveyors etc. which I feel the solar panels, as useless as they are, fit more into, don't have this suffix.

Reply:
Oh duh you're right, I'll get on that.
Is it just solar panels?

Reply:
yes